.github/workflows/tests.yaml

```diff
@@ -16,6 +16,13 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install Docker Compose v2
+        run: |
+          sudo mkdir -p /usr/local/lib/docker/cli-plugins/
+          sudo curl -SL https://github.com/docker/compose/releases/download/v2.2.3/docker-compose-linux-x86_64 -o /usr/local/lib/docker/cli-plugins/docker-compose
```

Comment:
That's it. With
```
sudo curl -SL https://github.com/docker/compose/releases/download/v2.2.3/docker-compose-linux-x86_64 -o /usr[/local]/bin/docker-compose
```
the test suite should work with absolute 0 changes. At least it did in my computer :woman_shrugging: :smile:

Reply:
BTW, Docker Compose v2 is now installed in Ubuntu images in GitHub Actions. I'll remove that step.